types.d.ts

```diff
@@ -1,8 +1,6 @@
 /* Any copyright is dedicated to the Public Domain.
  * http://creativecommons.org/publicdomain/zero/1.0/ */
 
-export as namespace bcd;
```

Comment:
I previously misunderstood what `export as namespace` meant.

I now know that this means export as a UMD global, which doesn’t exist.